demo/src/main/frontend/src/component/editUserInfo/EditUserInfo.js

```diff
@@ -0,0 +1,169 @@
+import { Button } from "react-bootstrap";
+import {icon} from 'bootstrap'
+import Navbar from "../navbar/Navbar";
+import "bootstrap-icons/font/bootstrap-icons.css";
+import { addUser } from "../../reducers/user/user";
+import { Link, useNavigate   } from 'react-router-dom';
+import { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import axios from "axios";
+import "./editUserInfo.css"
+
+function EditUserInfo(){
+
+     
+    const [name, setName] = useState();
+    const [PhoneNumber, setUserPhoneNumber] = useState();
+    const [password1, setPassword1] = useState();
+    const [password2, setPassword2] = useState();
+    const [data, setData] = useState();
+    const navigate = useNavigate();
+    let result = false
+
+
+   const dispatch = useDispatch();
+   const state = useSelector((state) => {
+    return {
+      user: state.userReducer,
+      token: state.userReducer.token
+    };
+  });
+
+   let id = state.user.user.id
+   let userToken = state.token
```

Comment:
any reason why these are let and not const? if you are not going to change the value then it should be const